rebasehelper/application.py

```diff
@@ -543,6 +546,10 @@ def build_packages(self):
                             if build_dict['rpm'] is None:
                                 return False
                     # Build finishes properly. Go out from while cycle
+
+                    # Do not store rebase-helper internal data to the results
+                    for opt in ['builds_nowait', 'build_tasks', 'builder_options']:
+                        build_dict.pop(opt)
```

Comment:
Just an idea, but wouldn't it be better to separate results from inputs?

Reply:
Maybe it is just a matter of a preference. Seeing output is tightly connected with the input parameters used, which can be useful. But I can hide the keys with the shown way.

Reply:
I agree with @thozza, IMO it shouldn't be visible in the output. But that doesn't mean it can't be part of `build_dict`.